distribution/smarthome/conf/things/demo.things

```diff
@@ -1,3 +1,3 @@
-yahooweather:weather:berlin [ location="638242" ]
-yahooweather:weather:losangeles "Los Angeles" [ location="2442047", unit="us", refresh=120 ]
+yahooweather:weather:berlin [ location=638242 ]
+yahooweather:weather:losangeles "Los Angeles" [ location=2442047, unit="us", refresh=120 ]
```

Comment:
Too late... have already resetted the branch... you will receive a further PR soon :)